### ISSUE TITLE
Incorrect adding widgets

### ISSUE THREAD

**Issue by cheack** [ducksboard/gridster.js#403]
Hi! 
Widget is added incorrect when i put string parameters to "add_widget" method e.g. 

<pre>gridster.add_widget(widget, "3", "3")</pre>

demo: http://jsfiddle.net/ekvfe/

**Comment by hrosenbauer** [comment]
Yes, because the API documentation states that size_x and size_y have to by of type Number, not String:
http://gridster.net/#add_widget_method
